tests/automation/disappearing_messages.spec.ts

```diff
@@ -18,207 +20,199 @@ import {
   waitForTextMessage,
 } from './utilities/utils';
 
-test.beforeEach(beforeAllClean);
+sessionTestThreeWindowsWithTwoLinked(
+  'Disappear after read 1:1',
+  ['Alice', 'Bob'],
```

Comment:
@burtonemily do we want those to be always the same to start with (and so I could just assume those in the fixtures)

Reply:
made those always the same usernames